components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift

```diff
@@ -57,6 +57,7 @@ class ButtonsSwiftAndStoryboardController: UIViewController {
     super.init(coder: aDecoder)
```

Comment:
This whole example is not actually running properly. May be related to issue #1156 . Maybe should have its own ticket. (IBOutlets aren't hooked up.)